frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCard/FeatureStrategyMenuCard.tsx

```diff
@@ -34,17 +34,30 @@ const StyledIcon = styled('div')(({ theme }) => ({
 }));
 
 const StyledDescription = styled('div')(({ theme }) => ({
-    fontSize: theme.fontSizes.smallBody,
+    fontSize: theme.typography.body2.fontSize,
```

Comment:
Same point about changing the base element here. If this is for text, then a `p` tag is better suited.